src/card.css

```diff
@@ -0,0 +1,113 @@
+.card {
```

Comment:
Your card should have padding all over the sides. It gives you a right inner space
And the - just use margin-bottom for card children to create a distance between them